src/Object/Pager.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Friendica\Object;
+
+/**
+ * A pager class.
+ */
+class Pager
```

Comment:
Do we need a whole class for this? This looks superfluous now that you moved all the display logic to RenderedPager.

Reply:
Why not?
- auto-calculate start
- check for boundaries
- bind together functional values

It's a simple class, but still useful I think

Reply:
I'll take this class for pagination of the backend search/match/contact requests in the near future :D